metals/src/main/scala/scala/meta/internal/metals/Doctor.scala

```diff
@@ -125,10 +130,14 @@ final class Doctor(
 
     def isMaven: Boolean = workspace.resolve("pom.xml").isFile
     def hint() =
-      if (isMaven)
-        "enable SemanticDB following instructions on the " +
-          "<a href=https://scalameta.org/metals/docs/build-tools/maven.html>Metals website</a>"
-      else s"run 'Build import' to enable code navigation."
+      if (isMaven) {
+        val website =
+          if (config.doctorFormat.value == "html")
```

Comment:
?
```suggestion
          if (config.doctorFormat.isHtml)
```

Reply:
_Makes `isHtml` method... and then doesn't use it._ 

Yes, ha, I'll use this.